src/quemb/molbe/solver.py

```diff
@@ -188,10 +189,11 @@ def from_user_input(cls, user_args: DMRG_ArgsUser, mf: RHF):
 
 @define(frozen=True)
 class SHCI_ArgsUser(UserSolverArgs):
-    hci_pt: Final[bool] = False
     hci_cutoff: Final[float] = 0.001
-    ci_coeff_cutoff: Final[float | None] = None
-    select_cutoff: Final[float | None] = None
+    hci_pt: Final[bool] = False
+    ci_coeff_cutoff: Final[float | None] = None  # TODO SOLVER
```

Comment:
As talked in person.

Personally, I would remove unused arguments. Unless you plan to implement them in the near future.